core/tests/protractor/editorFeatures.js

```diff
@@ -56,20 +62,22 @@ describe('Exploration history', function() {
     var COLOR_RENAMED_UNCHANGED = 'rgb(255, 215, 0)';
 
     // Check renaming state, editing text, editing interactions and adding state
-    editor.setStateName('first');
-    editor.setContent(forms.toRichText('enter 6 to continue'));
-    editor.setInteraction('NumericInput');
-    editor.addResponse('NumericInput', null, 'second', true, 'Equals', 6);
-    editor.moveToState('second');
-    editor.setContent(forms.toRichText('this is card 2'));
-    editor.setInteraction('Continue');
-    editor.setDefaultOutcome(null, 'final card', true);
-
+    explorationEditorPage.setStateName('first');
+    explorationEditorPage.setContent(forms.toRichText('enter 6 to continue'));
+    explorationEditorPage.setInteraction('NumericInput');
+    explorationEditorPage.addResponse(
+      'NumericInput', null, 'second', true, 'Equals', 6);
+    explorationEditorPage.moveToState('second');
+    explorationEditorPage.setContent(forms.toRichText('this is card 2'));
+    explorationEditorPage.setInteraction('Continue');
+    // explorationEditorPage.setDefaultOutcome(null, 'final card', true);
```

Comment:
Why comment? There should be no commented-out code in the final PR -- either uncomment it or remove it.

Reply:
Looks like that was left behind by mistake. Thanks for spotting it!